chore(docs): Add section about uptime (#158)
davidsonfellipe/awesome-wpo: README.md

- [beeswithmachineguns](https://github.com/newsapps/beeswithmachineguns) - A utility for arming (creating) many bees (micro EC2 instances) to attack (load test) targets (web applications).
- [k6](https://k6.io/) - An open source load testing tool built for developers. Easy to integrate into CI pipelines. Tests are written in ES6 JS and you can test APIs, microservices and sites using HTTP/1.1, HTTP/2 and WebSocket.

# Uptime

> - [Pingdom](https://pingdom.com) - A tool to get the uptime of your website (with probes from many location)

Reviewer: ```suggestion
- [Pingdom](https://pingdom.com) - A tool to get the uptime of your website (with probes from different locations)
```

- [UptimeRobot](https://uptimerobot.com) - Another uptime monitoring tool (with a generous free plan)
- [Nodown](https://nodown.io) - Also, a monitoring tool, but that focuses on feature testing

> - [BetterUptime](https://betteruptime.com) - A good website monitoring tool (bundling status page, indicent notification, ...)

Reviewer: ```suggestion
- [BetterUptime](https://betteruptime.com) - A good website monitoring tool (bundling status page, incident notification, ...)
```

> - [Hyperping](https://hyperping.com) - A good website monitoring tool (with a clean UI)

Reviewer: The domain name hyperping.com is for sale!

```suggestion
```

Reviewer: <img width="716" alt="Screen Shot 2022-02-06 at 9 25 20 PM" src="https://user-images.githubusercontent.com/381179/152715096-a3771306-90c0-46d9-bf7c-3e13efbba073.png">
